frontend/test/__support__/ui.tsx

```diff
@@ -72,6 +73,7 @@ export function renderWithProviders(
 
   if (withRouter) {
     Object.assign(reducers, { routing: routerReducer });
+    Object.assign(initialState, { routing });
```

Comment:
`initialState` cannot have `routing` key present when `routingReducer` is not used - `createStore` from redux will spam the console with warning about an extraneous key.